### DISCUSSION TITLE
Question: Why do some people add the /dev/net/tun device?

### DISCUSSION THREAD

**Discussion by simphide** [bubuntux/nordvpn#263] [Q&A] [answered]
Hi,

this is just a quick question since I have'nt found an answer yet: 
Why do some people add the /dev/net/tun device to the container? Why should you need that? Can the container work without that device? 

Thank you in advance!

**Comment by bubuntux** [comment, marked as answer]
it's not required, it's used for openvpn

**Reply by simphide** [reply]
So if you use NoryLynx you don't need to add the device?

**Reply by bubuntux** [reply]
correct
